synthetics/test_schema.go

```diff
@@ -166,133 +173,133 @@ func makeTestIPSchema(mode schemaMode) *schema.Schema {
 }
 
 func makeTestAgentSchema(mode schemaMode) *schema.Schema {
-	return makeNestedObjectSchema(mode, map[string]*schema.Schema{
+	return makeNestedObjectSchemaOptional(mode, map[string]*schema.Schema{
 		"target": {
 			Type:     schema.TypeString,
-			Optional: optionalOnCreate(mode),
+			Optional: true,
 			Computed: computedOnRead(mode),
 		},
 	})
 }
 
 func makeTestFlowSchema(mode schemaMode) *schema.Schema {
-	return makeNestedObjectSchema(mode, map[string]*schema.Schema{
+	return makeNestedObjectSchemaOptional(mode, map[string]*schema.Schema{
 		"target": {
 			Type:     schema.TypeString,
-			Optional: optionalOnCreate(mode),
+			Optional: true,
 			Computed: computedOnRead(mode),
 		},
 		"target_refresh_interval_millis": {
 			Type:     schema.TypeInt,
-			Optional: optionalOnCreate(mode),
+			Optional: true,
 			Computed: computedOnRead(mode),
 		},
 		"max_tasks": {
 			Type:     schema.TypeInt,
-			Optional: optionalOnCreate(mode),
+			Optional: true,
 			Computed: computedOnRead(mode),
 		},
 		"type": {
 			Type:     schema.TypeString,
-			Optional: optionalOnCreate(mode),
+			Optional: true,
 			Computed: computedOnRead(mode),
 		},
 	})
 }
 
 func makeTestSiteSchema(mode schemaMode) *schema.Schema {
-	return makeNestedObjectSchema(mode, map[string]*schema.Schema{
+	return makeNestedObjectSchemaOptional(mode, map[string]*schema.Schema{
 		"target": {
 			Type:     schema.TypeString,
-			Optional: optionalOnCreate(mode),
+			Optional: true,
 			Computed: computedOnRead(mode),
 		},
 	})
 }
 
 func makeTestTagSchema(mode schemaMode) *schema.Schema {
-	return makeNestedObjectSchema(mode, map[string]*schema.Schema{
+	return makeNestedObjectSchemaOptional(mode, map[string]*schema.Schema{
 		"target": {
 			Type:     schema.TypeString,
-			Optional: optionalOnCreate(mode),
+			Optional: true,
 			Computed: computedOnRead(mode),
 		},
 	})
 }
 
 func makeTestDNSSchema(mode schemaMode) *schema.Schema {
-	return makeNestedObjectSchema(mode, map[string]*schema.Schema{
+	return makeNestedObjectSchemaOptional(mode, map[string]*schema.Schema{
 		"target": {
 			Type:     schema.TypeString,
-			Optional: optionalOnCreate(mode),
+			Optional: true,
```

Comment:
`target` is required

Reply:
Done

Reply:
Does the `Optional: true` setting make the attribute required on create?

Reply:
No, Piotr missed that one and I didn't check that properly